test/e2e/flowaggregator_test.go

```diff
@@ -181,64 +195,66 @@ func checkRecordsForFlows(t *testing.T, data *TestData, srcIP string, dstIP stri
 	}
 	bandwidth := strings.TrimSpace(stdout)
 
-	// Adding some delay to make sure all the data records corresponding to iperf flow are received.
-	time.Sleep(collectorCheckDelay)
+	// Polling to make sure all the data records corresponding to iperf flow are received.
+	err = wait.Poll(250*time.Millisecond, collectorCheckTimeout, func() (bool, error) {
+		rc, collectorOutput, _, err := provider.RunCommandOnNode(masterNodeName(), fmt.Sprintf("kubectl logs --since=%v ipfix-collector -n antrea-test", time.Since(timeStart).String()))
+		if err != nil || rc != 0 {
+			return false, err
+		}
+		return strings.Contains(collectorOutput, srcIP) && strings.Contains(collectorOutput, dstIP), nil
+	})
+	assert.NoError(t, err, "IPFIX collector did not receive expected number of data records and timed out.")
```

Comment:
should this be `require.NoError`? If we don't receive the data records, should be bother with the validation after?

Reply:
makes sense.